index.js

```diff
@@ -104,29 +151,52 @@ exports.init = function (sbot, config) {
     if (rpc.id === sbot.id) return // ssb-client connecting to ssb-server
     if (isClient) {
       initialized.promise.then(() => {
-        const opts = { version: 3 }
-        const local = toPull.duplex(ebt.createStream(rpc.id, opts.version, true))
-        const remote = rpc.ebt.replicate(opts, (networkError) => {
-          if (networkError && getSeverity(networkError) >= 3) {
-            console.error('rpc.ebt.replicate exception:', networkError)
-          }
+        ebts.forEach(ebt => {
+          const format = ebt.name
+          const opts = { version: 3, format }
+          const local = toPull.duplex(ebt.createStream(rpc.id, opts.version, true))
+
+          // for backwards compatibility we always replicate classic
+          // feeds using existing replicate RPC
+          const replicate = (format === 'classic' ? rpc.ebt.replicate : rpc.ebt.replicateFormat)
+
+          const remote = replicate(opts, (networkError) => {
+            if (networkError && getSeverity(networkError) >= 3) {
+              console.error('rpc.ebt.replicate exception:', networkError)
+            }
+          })
+          pull(local, remote, local)
         })
-        pull(local, remote, local)
       })
     }
   })
 
-  function request (destFeedId, requesting) {
+  function findEBTForFeed (feedId, formatName) {
+    let ebt
+    if (formatName) { ebt = ebts.find(ebt => ebt.name === formatName) } else { ebt = ebts.find(ebt => ebt.isFeed(feedId)) }
```

Comment:
I put a comment here but it got swallowed after the lint (PS: what is the linter doing omg)

> By the way, when using this latest branch in ssb-replication-scheduler, I keep getting classic when the input is an index feed, but if I add .filter(ebt => ebt.name !== 'classic') then that fixes it.

Reply:
Yeah I'm not a super fan of those 1 liner tuborgs. There are several of them it made. The other one we should discuss :)